test/recipeRepository-test.js

```diff
@@ -1,27 +1,278 @@
+            "quantity": {
+              "amount": 1,
+              "unit": "tablespoon"
+            }
+          },
+          {
+            "name": "garlic",
+            "id": 11215,
+            "quantity": {
+              "amount": 1,
+              "unit": "clove"
+            }
+          },
+          {
+            "name": "grainy mustard",
+            "id": 1012046,
+            "quantity": {
+              "amount": 1,
+              "unit": "tablespoon"
+            }
+          },
+          {
+            "name": "maple syrup",
+            "id": 19911,
+            "quantity": {
+              "amount": 0.25,
+              "unit": "cup"
+            }
+          },
+          {
+            "name": "Miso Soybean Paste",
+            "id": 16112,
+            "quantity": {
+              "amount": 1,
+              "unit": "tablespoon"
+            }
+          },
+          {
+            "name": "pork chops",
+            "id": 10010062,
+            "quantity": {
+              "amount": 24,
+              "unit": "ounce"
+            }
+          },
+          {
+            "name": "salt and pepper",
+            "id": 1102047,
+            "quantity": {
+              "amount": 4,
+              "unit": "servings"
+            }
+          },
+          {
+            "name": "soy sauce",
+            "id": 16124,
+            "quantity": {
+              "amount": 1,
+              "unit": "tablespoon"
+            }
+          },
+          {
+            "name": "sriracha",
+            "id": 1016168,
+            "quantity": {
+              "amount": 1,
+              "unit": "tablespoon"
+            }
+          }
+        ],
+        "instructions": [
+          {
+            "number": 1,
+            "instruction": "Season the pork chops with salt and pepper and grill or pan fry over medium high heat until cooked, about 3-5 minutes per side. (If grilling, baste the chops in the maple dijon apple cider sauce as you grill.)Meanwhile, mix the remaining ingredients except the apple slices, bring to a simmer and cook until the sauce thickens, about 2-5 minutes.Grill or saute the apple slices until just tender but still crisp.Toss the pork chops and apple slices in the maple dijon apple cider sauce and enjoy!"
+          }
+        ],
+        "tags": [
+          "lunch",
+          "main course",
+          "main dish",
+          "dinner"
+        ]
+      }
+    ];
+
+    recipeRepository = new RecipeRepository(recipeTestData);
   });
 
-  it('Should have an array of all recipes', () => {
-    expect(recipeRepository.recipes).to.be.an('array');
+  it('should hold recipe data', () => {
+    
+    expect(recipeRepository.recipes).to.equal(recipeTestData);
   });
 
-  describe('findRecipe', () => {
-    it('Should be able to filter through its array by ingredients', () => {
-      expect(recipeRepository.findRecipe('yolk').length).to.equal(2);
-    });
+  it('should return recipes by tags', () => {
```

Comment:
There's a similar method in the User class that we might want to look at potentially creating a helper method for later if it's possible to reuse it. With that one, it's for filtering by tag in the User favorite recipes. This is more a note to possibly revisit that later.